app/models/answer.rb

```diff
@@ -1,4 +1,8 @@
 class Answer < ApplicationRecord
   belongs_to :user
   belongs_to :question
+
+  validates :user, presence: true
+  validates :question, presence: true
+  validates :content, length: { maximum: 100 }
```

Comment:
I think removing this restriction to allow the default maximum size for type 'text' (30,000 characters) is okay. For reference, the maximum number of characters allowed for type 'string' is 255. 
There may be a situation where university students ask questions that warrant a lengthy response for subjective questions such as an English analysis?